compiler/src/iree/compiler/Dialect/VM/Conversion/VMToEmitC/EmitCBuilders.h

```diff
@@ -18,9 +18,17 @@ namespace mlir {
 namespace iree_compiler {
 namespace emitc_builders {
 
+typedef struct StructField {
+  std::string type;
+  std::string name;
+} StructField;
+
 Value arrayElementAddress(OpBuilder builder, Location location, Type type,
                           IntegerAttr index, Value operand);
 
+void structDefinition(OpBuilder builder, Location location,
+                      StringRef structName, SmallVector<StructField> fields);
```

Comment:
```suggestion
                      StringRef structName, ArrayRef<StructField> fields);
```
always use array refs when you can - if you were std::move'ing the fields into storage then the vector is fine, but otherwise it's just introducing a copy on every call to the function that then just iterates over it.